lang/lang_en_fr.txt

```diff
@@ -94,10 +106,22 @@
 "Bed level correct"
 "Corr. niveau plateau"
 
+#MSG_BELTTEST c=17
+"Belt test        "
+"Test de courroie "
+
 #MSG_BED_LEVELING_FAILED_POINT_LOW c=20 r=4
 "Bed leveling failed. Sensor didnt trigger. Debris on nozzle? Waiting for reset."
 "Echec bed leveling. Capt. non declenche. Debris sur buse? En attente d'un reset."
 
```

Comment:
"Echec niveau lit, Capt. non declenche. Debris sur buse? En attente d'un reset

Reply:
`Echec niveau lit` translates to "bed level failed"(as in you failed a video game level centred around a bed). A better translation would be `Capt. non declenche pendant je nivele le plateau.`

Reply:
"capt. non declenche pendant que je nivele le plateau."

Reply:
Unless I miscounted, there is no space for a `que` there(4 row limit, 20 characters per row). I belive that the message is not greatly changed by missing it.

Reply:
> "capt. non declenche pendant que je nivele le plateau."

Use this for the 1st sentence. 
Long messages c=20 r>1 aren't a big issue to make here r=5 or 6 even saving space is also needed. We try keep it short and clear.